README.md

```diff
@@ -43,6 +43,9 @@ Contributers can build El-MAVEN on Windows, Ubuntu or Mac systems by following t
 - `pacman --force -Syu`
 - `pacman --force -Su`
 - `pacman --force -Sy base-devel msys2-devel mingw-w64-x86_64-toolchain mingw-w64-x86_64-qt5 mingw64/mingw-w64-x86_64-hdf5 mingw64/mingw-w64-x86_64-netcdf mingw64/mingw-w64-x86_64-boost msys/git mingw-w64-x86_64-curl`
+-  `Download OpenSSL package using https://indy.fulgan.com/SSL/openssl-1.0.2r-x64_86-win64.zip`
+- `Extract the contents of OpenSSL package in /c/msys64/mingw64/bin`
```

Comment:
Again this is an instruction not a shell command. So we can either have this:
> Extract the contents of OpenSSL package in `/c/msys64/mingw64/bin`

... or have an alternative unzip command such as:
`tar -xvf openssl-1.0.2r-x64_86-win64.zip /c/msys64/mingw64/bin`